examples/active-active/versions.tf

```
@@ -0,0 +1,16 @@
+terraform {
+  required_version = "~> 1.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.2"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.1"
+    }
+  }
+}
+provider "azurerm" {
```

Just adding a return between blocks for consistency.

```suggestion

provider "azurerm" {
```